bots/pr/src/main/java/org/openjdk/skara/bots/pr/AuthorCommand.java

```diff
@@ -105,14 +105,13 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
                     }
                 }
                 if (currAuthor.isEmpty()) {
-                    reply.println("There is no author set in this pull request.");
+                    reply.println("There is no overriding author set in this pull request.");
```

Comment:
I think "for" reads slightly better.
```suggestion
                    reply.println("There is no overriding author set for this pull request.");
```